app/Mage.php

```diff
@@ -703,7 +703,7 @@ public static function init($code = '', $type = 'store', $options = [], $modules
             } else {
                 self::$_app->init($code, $type, $options);
             }
-        } catch (Mage_Core_Model_Session_Exception $e) {
+        } catch (Mage_Core_Model_Session_Exception) {
             header('Location: ' . self::getBaseUrl());
             die;
         } catch (Mage_Core_Model_Store_Exception $e) {
```

Comment:
I do not understand the rationale of this PR. Why `$e` isn't remove here?

Reply:
I don't know. Think I have to report it to rector.

Reply:
Maybe it is passed to next catch-statement. I dont know. Ignore it for now?